go_mod/types.go

```diff
@@ -12,6 +12,6 @@ type goMod struct{}
 var _ common.GoModFile = &goModFile{}
```

Comment:
If you move this to _test file it's not going to be using space inside the final binary